source/base/Event.ooc

```diff
@@ -69,7 +63,14 @@ Event1: class <T> {
 	}
 	*/
```

Comment:
I think you should remove this big comment (And the one further down) while you're at it.